test/kernels/kernelsum.jl

```diff
@@ -18,4 +18,23 @@
     end
 
     test_params(k1 + k2, (k1, k2))
+
+    @testset "Type stability" begin
```

Comment:
```suggestion
    # Regression tests for https://github.com/JuliaGaussianProcesses/KernelFunctions.jl/issues/458
    @testset "Type stability" begin
```
just so that we know where things have come from